fio/run.sh

```diff
@@ -50,82 +48,81 @@ fi
 echo MODE: $MODE
 
 case $MODE in
-
-  "quick")
-    IOPS_FIO="iops-quick.fio"
-    BW_FIO="bandwidth-quick.fio"
-    LAT_FIO="latency-quick.fio"
-    ;;
-  "random-read-iops")
-    IOPS_FIO="iops-random-read.fio"
-    BW_FIO=""
-    LAT_FIO=""
-    ;;
-  "sequential-read-iops")
-    IOPS_FIO="iops-sequential-read.fio"
-    BW_FIO=""
-    LAT_FIO=""
-    ;;
-  "random-read-bandwidth")
-    IOPS_FIO=""
-    BW_FIO="bandwidth-random-read.fio"
-    LAT_FIO=""
-    ;;
-  "sequential-read-bandwidth")
-    IOPS_FIO=""
-    BW_FIO="bandwidth-sequential-read.fio"
-    LAT_FIO=""
-    ;;
-  "random-read-latency")
-    IOPS_FIO=""
-    BW_FIO="latency-random-read.fio"
-    LAT_FIO=""
-    ;;
-  "sequential-read-latency")
-    IOPS_FIO=""
-    BW_FIO="latency-sequential-read.fio"
-    LAT_FIO=""
-    ;;
-  "random-write-iops")
-    IOPS_FIO="iops-random-write.fio"
-    BW_FIO=""
-    LAT_FIO=""
-    ;;
-  "sequential-write-iops")
-    IOPS_FIO="iops-sequential-write.fio"
-    BW_FIO=""
-    LAT_FIO=""
-    ;;
-  "random-write-bandwidth")
-    IOPS_FIO="bandwidth-random-write.fio"
-    BW_FIO=""
-    LAT_FIO=""
-    ;;
-  "sequential-write-bandwidth")
-    IOPS_FIO="bandwidth-sequential-write.fio"
-    BW_FIO=""
-    LAT_FIO=""
-    ;;
-  "random-write-latency")
-    IOPS_FIO=""
-    BW_FIO="latency-random-write.fio"
-    LAT_FIO=""
-    ;;
-  "sequential-write-latency")
-    IOPS_FIO=""
-    BW_FIO="latency-sequential-write.fio"
-    LAT_FIO=""
-    ;;
-  "full" | "")
-    IOPS_FIO="iops.fio"
-    BW_FIO="bandwidth.fio"
-    LAT_FIO="latency.fio"
-    ;;
-
-  *)
-    echo "ERROR: unknown mode"
-    exit 1
-    ;;
+    "quick")
+        IOPS_FIO="iops-quick.fio"
+        BW_FIO="bandwidth-quick.fio"
+        LAT_FIO="latency-quick.fio"
+        ;;
+    "random-read-iops")
+        IOPS_FIO="iops-random-read.fio"
+        BW_FIO=""
+        LAT_FIO=""
+        ;;
+    "sequential-read-iops")
```

Comment:
Do we still need to retain sequential iops/latency related options here? They are already removed from the above scripts

Reply:
I think NO. We can remove them.